src/overlays/gamestates/ovl_title/z_title.c

```diff
@@ -91,7 +91,7 @@ void ConsoleLogo_Draw(ConsoleLogoState* this) {
     Gfx_SetupDL_39Opa(this->state.gfxCtx);
     gDPPipeSync(POLY_OPA_DISP++);
     gDPSetCycleType(POLY_OPA_DISP++, G_CYC_2CYCLE);
-    gDPSetRenderMode(POLY_OPA_DISP++, G_RM_XLU_SURF2, G_RM_OPA_CI | CVG_DST_WRAP);
+    gDPSetRenderMode(POLY_OPA_DISP++, G_RM_PASS, G_RM_XLU_SURF2 | CVG_DST_WRAP);
```

Comment:
`G_RM_XLU_SURF2 | CVG_DST_WRAP` does `CVG_DST_FULL | CVG_DST_WRAP = 0x200 | 0x100 = CVG_DST_SAVE` :sweat_smile: 

```suggestion
    gDPSetRenderMode(POLY_OPA_DISP++, G_RM_PASS, G_RM_CLD_SURF2);
```